### ISSUE TITLE
asiapool.electroneum.com hasn't paid out since 1/3

### ISSUE THREAD

**Issue by Fritticys** [electroneum/electroneum-pool#140]
Just as the subject says.  I have 40+ coin waiting for payout and nothings triggered for any payments since the 3rd.  I had no idea where to bring this to anyones attention, so here seemed appropriate, appologies if this isn't the correct venue.

**Comment by SunnyAlam890** [comment]
I am having the same issue Asia pool. My pending balance is increasing when i mine, but the pending balance is not being transferred to Paid Balance. My pending balance is now at $56, but not yet paid.
Anyone know how to fix this? Thanks.

**Comment by Fritticys** [comment]
Update: a payment went out this morning, I was not included in it, however at least the clogs unclogging

**Comment by SunnyAlam890** [comment]
Thats good, do you see your total Paid Balance increase? 
I havent seen any change in my Pending balance, its still stuck at 54 ETN.

My offline wallet: etnk5TZpnHRCA42Qj2Mk3rD7GQJDc9NxfdkE76TNtKjrR3EZwCqUFqHa6C1SPst4rC1M8EB4TS7zPV6Um1NrFCFM9993d53tWK

**Comment by Fritticys** [comment]
I haven't been mining since I realized payments stopped, took the opportunity to test a few other pools.  I'd like to continue mining here.  I started back up to test to see if my unpaid balance is frozen like yours is.  Haven't found a block since I switched back.

**Comment by agussang** [comment]
no payment from asiapool.electroneum.com, i think this coin will be scam coin. so fucking idiot thats official pool no payment. thats pool has stopped payment from 1/3/2018, and pool start fake payment on 1/7/2018 with transaction hash 15dd2b8ddbeb04f5facd232cf3e0d088fee75895d7b837b97a14b2af1c1ec667, but can't found https://blockexplorer.electroneum.com/tx/15dd2b8ddbeb04f5facd232cf3e0d088fee75895d7b837b97a14b2af1c1ec667

hahaha, let's banned this coin

**Comment by ariadhi** [comment]
@agussang only one TX Hash sent today, and yes it is not valid. My 71 ETN still pending !!!!!
I though this is official pool and everyone should be aware about this.

I have to stop mining and move to other pool, otherwise I will move to other coin which is more promising.

**Comment by agussang** [comment]
@ariadhi may be other pool will be stopped bro, look for https://eupool.electroneum.com/ thats payment has been stopped, yes this official pool and stopped payment, everyone must be aware with this coin. i was contact helpdesk but nothing answer.

**Comment by ariadhi** [comment]
Now they stop my mining, I only get 0h/s while my mining still working.
Gezz... This is not acceptable.

![2018-01-08_14-12-26](https://user-images.githubusercontent.com/35096727/34661831-cfc84bd8-f47e-11e7-8648-ab336189979a.jpg)

**Comment by agussang** [comment]
@ariadhi ahhaa,, shit pool

**Comment by ariadhi** [comment]
Now they completely stopped, the page show overloaded..  I already moved to other coin xmr.

**Comment by agussang** [comment]
@ariadhi just waste time minning this coin.

**Comment by SunnyAlam890** [comment]
Even the EU pool or any of the Major pools, all of them having issues with payment.
Dont know about the other pools like;
http://electroneum.hashparty.io
http://pool.electroneumcharts.com
http://etnhash.com

I have opened ticket to their support team, no response so far! ETN looked so promising

**Comment by agussang** [comment]
If they unsupport with their official pool, so for what he make coin. Yeah i know other pool, nanopool opened ETN.  But this probelm on they official pool (OFFICIAL), u know what i mean with OFFICIAL

**Comment by Fritticys** [comment]
I think you guys are jumping the gun on this one.  The uspool is still functioning perfectly fine, so is the other major pool i mine from.  Seems like they are having a problem with asia and eu, but if they were going to scam out, don't you think they would have dropped uspool aswell?  I could be wrong, but doesn't hurt to me optimistic.

**Comment by alesha17** [comment]
I also get the same thing as you guys I get 71 etn pending and still growing after I realized there was an impression with all this I instead get a pool with a setatus overload what is this whether the pool official wants to deceive us this should not happen because it will reduce pamot etn to exis in the world

**Comment by ariadhi** [comment]
I agree with @alesha17 and I believe they are doing something to fix the issue right now, at least they can explain to us what really happened and whatever the reasons will make us more confidence, well.... let's wait and see for couples of days.

**Comment by SunnyAlam890** [comment]
Yeah its true that we should not jump the gun, asia pool server is going under maintenance due to overload. At least they are working on letting us know whats going on. Puts my mind at ease at least. Lets see how it turns out before we start mining ETN again

**Comment by agussang** [comment]
Now, they kill uspool

**Comment by alesha17** [comment]
it looks like we should close in the hope of paying the results of our mining in asiapool, I see the web has been diverted to electroneum.com
why the official does not tell us "about the problem" is really disappointed

**Comment by ariadhi** [comment]
I've been losing my interestingly about etn since now, hopefully not all. Moving to another alternative for mining, I'll check for few next days, maybe week whether or not the services going back to normal. Truly they are very prospective and I have good feeling on that. 👍😀😊😍

**Comment by ariadhi** [comment]
Well I think another people trying to explain something to Etn dev, there are something wrong in the blockchain. You guys take a look in this link that might little answer for our questions mark

https://github.com/electroneum/electroneum-pool/issues/145

**Comment by agussang** [comment]
Today, all pool DIE

**Comment by SunnyAlam890** [comment]
wow, what is happening. Thank god, i stopped mining after seeing my balance stuck at asiapool. But why isnt there any news on this? or if they are working to fix it or WHATEVER?

**Comment by incyi** [comment]
I have the same issue at eupool.electroneum.com

**Comment by ariadhi** [comment]
Guys, check this one 

https://m.facebook.com/story.php?story_fbid=1941250249470277&id=1850733001855336&ref=page_internal

**Comment by alesha17** [comment]
of the fb tread you provide, hoping it is true they will pay for all the pending coin, which has been painstakingly miners struggling to find

**Comment by SunnyAlam890** [comment]
thanks @ariadhi Finally some info from them on that FB thread. Yeah i also hope they pay out our pending ETN balance. Though i am still reluctant to go back mining ETN for now, until we can know that their payment system works properly.

**Comment by Fritticys** [comment]
Still waiting on pending balance. Not too worried just updating those who read this. I have 55 on Asia and 10 on us

**Comment by dyablohunter** [comment]
The best pool is the pool with the best infrastructure. That is http://electroneum.vip - hosted on Amazon EC2 servers that can scale to infinite, they run on the backbone of the internet in Ohio, US. Very warm community there on Discord as well.

**Comment by SunnyAlam890** [comment]
Well i still havent been paid on my pending balance on my Offline wallet. According to their facebook post, they are going to pay us on our pending balance, but dont know how long that might take.
I will wait until that time to start mining again.

**Comment by halbi** [comment]
I have also the same problem. I mined around 35 ETN on Asia pool. However, coins did not be transferred to my wallet. I wrote about this situation to the ETN support.

**Comment by briancfalls** [comment]
I was just finally able to access my wallet, zero payout from EU pool and zero payout from the asia pool, How come no one is talking about how they ripped us all off? It is now Feb 2nd, 2018

**Comment by ariadhi** [comment]
And I dont received my 71 etn yet, it's not like they said before that they will send all pending payment.
